lib/masamune/abstract_syntax_tree/node_helper.rb

```diff
@@ -0,0 +1,90 @@
+module Masamune::AbstractSyntaxTree::NodeHelper
+  # The source code of Prism nodes can be retrieved by calling #slice.
+  # However, the output tends to vary from node to node, and other methods
+  # like `name`, `message`, etc. are available in Prism nodes which means you
+  # have to know exactly which method to call the get the results you want.
+  #
+  # The `token` method simplifies all of this, and just gives you the
+  # variable, method name, etc. in String form by just calling `token`.
+  def token
```

Comment:
Is `token` the right name? My understanding of ASTs is that tokens would more refer to `CommentNode` etc.

This seems more like it should be named `content`, `name` or `value`.

Reply:
I could be wrong on the precise meaning of the terminology, but I wanted one word that we could refer to to simply get the String we want without having to think too much if it's a `name`, `value`, etc.

We can still technically access all of the other methods from Prism nodes like `message`, `value`, etc., but I think this will be easier development-wise to just refer to the `token`.

Say we have a `LocalVariableWriteNode`:
```ruby
code = <<~CODE
  x = 1
CODE
p variable_node = Masamune::AbstractSyntaxTree.new(code).variables.first
```

```
=> @ LocalVariableWriteNode (location: (1,0)-(1,5))
├── name: :x
├── depth: 0
├── name_loc: (1,0)-(1,1) = "x"
├── value:
│   @ IntegerNode (location: (1,4)-(1,5))
│   └── flags: decimal
└── operator_loc: (1,2)-(1,3) = "="
```

`#slice` and `#name` give us the following:
```
[1] pry(#<TestMasamune>)> variable_node.slice
=> "x = 1"
[2] pry(#<TestMasamune>)> variable_node.name
=> :x
```

I chose to implement `token` so we could just get a simple string, whether it's just a variable name, method name, symbol value, String, etc.

Reply:
I'm completely with you on the implementation, I just think naming it `token` blurs the line between AST concept names in a way that makes it tougher to understand.

I'm pushing a potential refactor in a bit where I've renamed this `token_value`, which I think is clearer.

Reply:
Resolved in #61